Formula/chakra.rb

```diff
@@ -12,23 +12,10 @@ class Chakra < Formula
   end
 
   depends_on "cmake" => :build
-  depends_on "icu4c" => :optional
 
   def install
-    # Build fails with -Os default
-    # Upstream issue from 26 Jan 2016 https://github.com/Microsoft/ChakraCore/issues/2417
-    # Fixed in master https://github.com/obastemur/ChakraCore/commit/cda81f4
-    ENV.O3
-
-    args = ["--static"]
-    if build.with? "icu4c"
-      args << "--icu=#{Formula["icu4c"].opt_include}"
-    else
-      args << "--no-icu"
-    end
-    system "./build.sh", *args
-
-    bin.install "BuildLinux/Release/ch" => "chakra"
+    system "./build.sh", "--lto-thin", "--static", "--embed-icu", "-j=#{ENV.make_jobs}", "-y"
```

Comment:
@obastemur That's because you don't want `=> :build` which means the dependency is build-time only. To protect against erroneous `=> :build` we uninstall them before running the test block!